documentation/feature/fcs.md

```diff
@@ -6,25 +6,31 @@ group: feature
 parent: wiki
 ---
 
-## Overview
+## 1. Overview
 
-### Fire control system
+### 1.1 Fire control system
 Offers a custom fire control system for tanks and helicopters. It enables engaging stationary and moving targets. 
 
-### Manual lasing targets
+### 1.2 Manual lasing targets
 Changes the default rangefinders, including those in vehicles, to require manual lasing.
 
-### Air burst ammunition
+### 1.3 Air burst ammunition
 Anti air cannons can now use airburst ammunition. It will explode on the FCS' zeroed in range.
 
 
-## Usage
+## 2. Usage
 
-To engage moving targets, place the crosshair on the enemy vehicle and press and hold tab. Follow the moving target with your crosshair for about 2 seconds and release tab. The optic will now be adjusted sideways to ensure a hit.
+### 2.1 Engaging moving targets
 
-To use manual lasing, place the crosshair on the object to range and press and hold tab.
+- Place the crosshair on the enemy vehicle.
+- Press and hold <kbd> tab </kbd> (by default) and follow the target for about 2 seconds.
+- The optic is now adjusted sideways to ensue a hit.
```

Comment:
after following the target for about a second you need to release tab to adjust the optic.